BangazonAPI/Controllers/CustomersController.cs

```diff
@@ -0,0 +1,336 @@
+﻿/*
+    Author: Jeremiah Pritchard
+    Purpose: API Controller that allows a client to: 
+            GET all Customers  from DB, 
+            GET a single Customer, 
+            POST a new Customer to the DB, 
+            PUT (edit) and existing Customerin the DB, and 
+            DELETE a Customer from the DB,
+            User can also query names of Customers with q= and _include=payment or products
+                will show payment types or products in information presented.
+ */
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using Dapper;
+using Microsoft.AspNetCore.Http;
+using BangazonAPI.Models;
+using BangazonAPI.Data;
+
+namespace BangazonAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomersController : ControllerBase
+    {
+        private readonly IConfiguration _config;
+        public CustomersController(IConfiguration config)
+        {
+            _config = config;
+        }
+        public IDbConnection Connection
+        {
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+        // GET api/customers?q=Taco
+        [HttpGet]
+        public async Task<IActionResult> Get(string q, string _include)
+        {
+            string sql;
+            if (_include != null && _include.Contains("payments"))
+            {
+                sql = @"
+            SELECT
+                c.Id,
+                c.FirstName,
+                c.LastName,
+                p.Id,
+                p.CustomerId,
+                p.Name    
+            FROM Customer c
+            JOIN PaymentType p ON p.CustomerId = c.Id          
+            WHERE 1=1
+            ";
+                if (q != null)
+                {
+                    string isQ = $@"
+                    AND c.FirstName LIKE '%{q}%'
+                    OR c.LastName LIKE '%{q}%'
+                ";
+                    sql = $"{sql} {isQ}";
+                }
+            }
+            else if (_include != null && _include.Contains("products"))
```

Comment:
When `_include=products` is specified and a customer has not created any products, that customer is not being returned. I would expect to see the customer with an empty list of products.